scheduler/scheduler-api/build.gradle

```diff
@@ -13,6 +13,7 @@ dependencies {
     compile project(':common:common-api')
 
     compile 'org.apache.commons:commons-lang3:3.5'
+    compile group: 'com.zaxxer', name: 'HikariCP', version: '2.3.2'
```

Comment:
We can add this in the Parent Bom project

Reply:
scheduling don't use parent-bom, (it is far too complicated to do so)

Reply:
ok